### PR TITLE
chore: export fonts and font stylesheets

### PR DESCRIPTION
**Pull request by jrschumacher**

### CODE REVIEW

File: config/rollup.config.js [hunk 1]
```diff
@@ -40,8 +38,10 @@ module.exports = {
     reactSvg(),
     copy({
       targets: [
-        { src: 'lib/styles/build/**/*.{css,js}', dest: 'dist' }
-      ]
-    })
+        { src: 'lib/styles/build/**/*.{css,js}', dest: 'dist' },
+        { src: 'lib/styles/typography/fonts/', dest: 'dist' },
+        { src: 'lib/styles/typography/css/*.css', dest: 'dist/font-style' },
```
Comment (avkvirtru) [inline config/rollup.config.js:43]: Maybe we can simplify the structure of `lib/styles/typography` in the near future? I was hoping for something like a **fonts.css** that we can add to `dist` for easier importing. 

This works for now but is very specific when importing elsewhere (like the example app):

```html
<link 
  rel="stylesheet" 
  href="https://unpkg.com/virtuoso-design-system@latest/dist/font-style/open-sans.css">
<link 
  rel="stylesheet" 
  href="https://unpkg.com/virtuoso-design-system@latest/dist/font-style/raleway.css">
```

Comment (avkvirtru) [reply config/rollup.config.js:43]: vs. a one-liner like:
```html
<link 
  rel="stylesheet" 
  href="https://unpkg.com/virtuoso-design-system@latest/dist/fonts.css">
```

File: config/rollup.config.js [hunk 2]
```diff
@@ -25,9 +25,7 @@ module.exports = {
   external: ['react', 'prop-types'],
   plugins: [
     alias({
-      entries: [
-        { find: '@', replacement: path.join(__dirname, '..', 'lib') }
-      ],
+      entries: [{ find: '@', replacement: path.join(__dirname, '..', 'lib') }],
```
[the remark this answers is not in the document]
Comment (jrschumacher) [reply config/rollup.config.js:28]: Yea it's eslint.

File: config/rollup.config.js (hunk 1 above)
Comment (jrschumacher) [reply config/rollup.config.js:43]: Per our convo, resolving.

**Pull request merged**